src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -42,36 +52,101 @@ Our AI has been trained to communicate with engineers using common questions and
 
 To enable NRAI in the NRQL console: 
 
 1. Go to the user menu at the bottom left of the New Relic UI.
 2.  Click the **Features** tab.
 3. Scroll down to the **New Relic AI** section and click **Enable**.
 
-## Sample questions [#sample-questions] 
+### Sample questions [#sample-questions] 
 
 You can chat with our AI to better understand every aspect of your data but if you're struggling with where to begin, here are some prompts. 
 
-### Get started with New Relic 
-Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
+<Tabs>
+    <TabsBar>
+    <TabsBarItem id="1">Get started with New Relic</TabsBarItem>
+    <TabsBarItem id="2">Improve response time</TabsBarItem>
+    <TabsBarItem id="3">Take observability to the next level</TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+
+    <TabsPageItem id="1`">
+    Our AI chat experience can also help get you started using New Relic. If you need help organizing your data and setting up alerts, NRAI can act as a guide throughout your onboarding process. Ask our AI questions like:
 
 * How can I use Terraform to create dashboards? 
 * How can I improve my alerts setup?
 * How can I instrument AWS?
+    </TabsPageItem>
 
-### Improve response time
-If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
+    <TabsPageItem id="2">
+    If you want to get to the root cause of latency issues for your app, here are some good questions to ask:
 
 * What are some recent incidents in my system?
 * Which infrastructure hosts are showing signs of degradation over the past hour?
+    </TabsPageItem>
 
-### Take observability to the next level 
-Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
+    <TabsPageItem id="3">
+    Our AI can also be your partner in observability maturity. If you're responsible for ensuring that your app can handle spikes for a big e-commerce event like Black Friday, it's important to understand the state of your system so you can improve it _ahead of time_. Ask our AI:
 
 * How can I improve the performance of my infrastructure?
 * Were unique user sessions up or down last week compared to the week before?
+    </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+### Get help analyzing error logs [#errors]
```

Comment:
The headers in this doc are a bit weird. Since there is quite a bit of content on the doc, it's a bit jarring to have so many h3s in the doc. I'd toy around with fixes (or don't, this is a small issue) but I think just making most of them h2s is fine.